<?xml version="1.0"?>
<launch>
  
  <node name="map_server" pkg="map_server" type="map_server" args="$(find navigation)/maps/map.yaml" />

  <node name="amcl"
        pkg="amcl"
        type="amcl"
        clear_params="true"
        output="screen">
    <rosparam command="load" file="$(find navigation)/config/amcl.yaml"/>
  </node>

  <!--node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
    <param name="serial_port"         type="string" value="/dev/rplidar"/>  
    <param name="serial_baudrate"     type="int"    value="256000"/>
    <param name="frame_id"            type="string" value="lidar_link"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="frame_id"            type="string" value="laser"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="output_scan"         type="string" value="/up_scan"/>
  </node-->

</launch>
